service/frontend/dcRedirectionPolicy_test.go

```diff
@@ -330,40 +329,6 @@ func (s *selectedAPIsForwardingRedirectionPolicySuite) TestGetTargetDataCenter_G
 	s.Equal(2*len(selectedAPIsForwardingRedirectionPolicyWhitelistedAPIs), alternativeClustercallCount)
 }
 
-func (s *selectedAPIsForwardingRedirectionPolicySuite) TestGetTargetDataCenter_GlobalDomain_Enforce_Forwarding_AlternativeClusterToCurrentCluster() {
```

Comment:
why is test deleted?

Reply:
Because YARPC doesn't provide a way to test -- no way to put this CallOptions into context ourselves. It's done internally.